pkg/helm/actions/config.go

```diff
@@ -11,6 +11,7 @@ import (
 )
 
 var settings = cli.New()
+var k8sInClusterCA = "/var/run/secrets/kubernetes.io/serviceaccount/ca.crt"
```

Comment:
This shouldn't be hard-coded here. We have logic to read this CA file when running in-cluster in main.go.